jib-core/src/main/java/com/google/cloud/tools/jib/filesystem/AbsoluteUnixPath.java

```diff
@@ -98,6 +98,42 @@ public AbsoluteUnixPath resolve(RelativeUnixPath relativeUnixPath) {
     return new AbsoluteUnixPath(newPathComponents.build());
   }
 
+  /**
+   * Resolves this path against another relative path (by the name elements of {@code
+   * relativePath}).
+   *
+   * @param relativePath the relative path to resolve against
+   * @return a new {@link AbsoluteUnixPath} representing the resolved path
+   */
+  // TODO: Add test.
```

Comment:
I think we should add tests in this PR, unless you will add these method in another PR.

Reply:
Oops, I've already added tests and forgot to take this out.